src/main/java/ch/uzh/ifi/hase/soprafs23/service/GameService.java

```diff
@@ -46,9 +46,15 @@ public class GameService {
     @Autowired
     public GameService(@Qualifier("gameRepository") GameRepository gameRepository, JobScheduler jobScheduler,
             LobbyService lobbyService) {
+        this(gameRepository, jobScheduler, lobbyService, new ImgflipClient());
+    }
+
+    public GameService(GameRepository gameRepository, JobScheduler jobScheduler,
```

Comment:
this constructor acts as another way to make a new instance of this, but crucially allows us to mock the "IMemeApi" in the tests.